src/pages/[platform]/build-a-backend/auth/modify-resources-with-cdk/index.mdx

```diff
@@ -114,3 +116,33 @@ cfnUserPool.enabledMfas = [...(cfnUserPool.enabledMfas || []), "EMAIL_OTP"]
 {/* token validity */}
 {/* BYO custom idp construct */}
 {/* extend auth/unauth roles */}
+
+### Override Cognito UserPool to enable passwordless sign-in methods
+
+You can modify the underlying Cognito user pool resource to enable sign in with passwordless methods. [Learn more about passwordless sign-in methods](/[platform]/build-a-backend/auth/concepts/passwordless/).
+
+```ts title="amplify/backend.ts"
+import { defineBackend } from "@aws-amplify/backend"
+import { auth } from "./auth/resource"
+
+const backend = defineBackend({
+  auth,
+})
+
+const { cfnResources } = backend.auth.resources;
+const { cfnUserPool, cfnUserPoolClient } = cfnResources;
+
+cfnUserPool.addPropertyOverride(
+	'Policies.SignInPolicy.AllowedFirstAuthFactors',
+	['PASSWORD', 'WEB_AUTHN', 'EMAIL_OTP', 'SMS_OTP']
+);
+
+cfnUserPoolClient.explicitAuthFlows = [
+	'ALLOW_REFRESH_TOKEN_AUTH',
+	'ALLOW_USER_AUTH'
+];
+
+/* Needed for WebAuthn */
+cfnUserPool.addPropertyOverride('WebAuthnRelyingPartyID', '<RELYING_PARTY>');
+cfnUserPool.addPropertyOverride('WebAuthnUserVerification', 'preferred');
```

Comment:
Do we need to update `UserPoolTier` here as well? Or is this not necessary?

`cfnUserPool.addPropertyOverride('UserPoolTier', 'PLUS');`